dags/dbt_advanced.py

```diff
@@ -50,15 +49,21 @@ def make_dbt_task(node, dbt_verb):
         dbt_task = BashOperator(
             task_id=node_test,
             bash_command=f"""
-            cd {DBT_DIR} &&
-            dbt {GLOBAL_CLI_FLAGS} {dbt_verb} --target prod --models {model}
+            dbt {GLOBAL_CLI_FLAGS} {dbt_verb} --target dev --models {model} \
+            --profiles-dir {DBT_PROJECT_DIR} --project-dir {DBT_PROJECT_DIR} 
             """,
             dag=dag,
         )
     return dbt_task
 
-data = load_manifest()
 
+dbt_seed = BashOperator(
```

Comment:
Is it possible to add a similar comment here like what's in the `dbt_basic` and `elt` DAG files to really hammer home that this task is really for demo purposes only?

Reply:
Added the comment